aiochsa/types.py

```diff
@@ -20,18 +20,18 @@
 class BaseType(Generic[PyType, JsonType]):
     __slots__ = ()
 
-    py_type: Optional[Type[PyType]] = None
+    py_type: Type[PyType]
 
     @classmethod
     def escape(cls, value: PyType, escape: Callable) -> str:
         return str(value)
 
-    def from_json(self, value: JsonType) -> PyType:
-        return self.py_type(value)
+    def from_json(self, value: JsonType) -> Optional[PyType]:
+        raise NotImplementedError()
 
     @classmethod
     def to_json(cls, value: PyType, to_json: Callable) -> JsonType:
-        return value
+        raise NotImplementedError()
```

Comment:
mypy forces me to duplicate blanket implementation across all subclasses. Is there a way to return default implementation here without just suppressing error with `ignore`?

Reply:
`cast()`?

Reply:
Both. I don't see how it can solve this case, and extra call here would seriously impact performance.